dsb-gui/build.gradle

```diff
@@ -42,3 +45,19 @@ shadowJar {
         exclude 'junit:junit:4.12'
     }
 }
+
+task genConfigProperties {
+    doLast {
+        Path configPath = sourceSets.main.output.resourcesDir.toPath()
+        Files.createDirectories(configPath)
+
+        File configFile = new File(sourceSets.main.output.resourcesDir, "/ds3_cli.properties")
+        configFile.withWriter{out ->
+            out.writeLine("version="+version)
+            out.writeLine("build.date="+new Date().toString())
+        }
+    }
+}
+
+shadowJar.dependsOn genConfigProperties
```

Comment:
This should happen on the jar task.  I'd verify that the jar task is called when the shadowJar is created, but it should be.